src/eslint-config-provider.ts

```diff
@@ -70,14 +70,20 @@ export class ESLintConfigProvider implements ConfigProvider {
     return this.factory.getConfigArrayForFile(fileName).extractConfig(fileName);
   }
 
-  private resolveESLintIntrinsicConfigPath(name: "eslint-all" | "eslint-recommended") {
-    let ret: string | undefined = undefined;
+  private resolveESLintIntrinsicConfigPath(name: "eslint-all" | "eslint-recommended"): string | undefined {
     try {
-      const fragments = require.resolve("eslint").split("node_modules/eslint");
-      ret = [...fragments.slice(0, fragments.length - 1), `/conf/${name}.js`].join("node_modules/eslint");
-    } catch (e: any) {
-      this.log(e);
+      // Config js files move to @eslint/js package sinse ESLint v8.35.
+      // https://github.com/eslint/eslint/pull/16844
+      return require.resolve(`@eslint/js/src/configs/eslint-${name}`);
```

Comment:
Should `eslint-${name}` just be `${name}` ?

Reply:
Thx 😂